ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -62,17 +63,14 @@ class CommandBarButtonGroupView: UIView {
 
     private func applyInsets() {
         if #available(iOS 15.0, *) {
-            buttons.first?.configuration?.contentInsets.leading += LayoutConstants.leftRightBuffer
-            buttons.last?.configuration?.contentInsets.trailing += LayoutConstants.leftRightBuffer
+            buttons.first?.configuration?.contentInsets.leading += CommandBarButtonGroupView.leftRightBuffer
+            buttons.last?.configuration?.contentInsets.trailing += CommandBarButtonGroupView.leftRightBuffer
         } else {
-            buttons.first?.contentEdgeInsets.left += LayoutConstants.leftRightBuffer
-            buttons.last?.contentEdgeInsets.right += LayoutConstants.leftRightBuffer
+            buttons.first?.contentEdgeInsets.left += CommandBarButtonGroupView.leftRightBuffer
+            buttons.last?.contentEdgeInsets.right += CommandBarButtonGroupView.leftRightBuffer
         }
     }
 
-    private struct LayoutConstants {
-        static let cornerRadius: CGFloat = 8.0
-        static let buttonPadding: CGFloat = 2.0
-        static let leftRightBuffer: CGFloat = 2.0
-    }
+    private var tokenSet: CommandBarTokenSet
+    private static let leftRightBuffer: CGFloat = 2.0
```

Comment:
Maybe keep this within a Constants struct to follow the Microsoft Swift Style guidelines?

Reply:
the buffer should technically be part of the tokenset's private static const like other fluent2 controls